src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -133,26 +133,47 @@
           </v-row>
         </template>
 
-        <div
-          v-if="minimumThreeDirectorError"
+        <MessageBox
+          v-if="hasAttemptedSubmission && minimumThreeDirectorError"
           id="minimum-three-director-error"
-          class="my-6"
+          class="mt-6"
+          color="red"
         >
-          <p class="error-text">
-            The business type cannot be changed. A Community Contribution Company requires a minimum of three directors.
+          <p>
+            <v-icon
+              color="red"
+              class="error-icon"
+            >
+              mdi-alert
+            </v-icon>
+            <strong class="pl-2 pt-6">Update directors</strong>
```

Comment:
I wonder if using [HTML Semantic Elements](https://www.w3schools.com/html/html5_semantic_elements.asp) would be a good idea here to improve accessibility? For example, this is definitely a header, and the body below is an article (or section).